lms/envs/common.py

```diff
@@ -735,21 +735,6 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 USERNAME_PATTERN = r'(?P<username>{regex})'.format(regex=USERNAME_REGEX_PARTIAL)
 
 
-############################## HEARTBEAT ######################################
```

Comment:
nitpick: could we just edit the one line instead of moving this whole block?

Reply:
The new line must be after HIGH_PRIORITY QUEUE definition. I could create it after it, outside HEARTBEAT section, but other routing keys are together with their related settings ( eg GRADES_DOWNLOAD_ROUTING_KEY and BULK_EMAIL_ROUTING_KEY). I tried to be consistent.

Reply:
Ah of course. :)
That wasn't in the diff and I didn't consider it.
This is definitely necessary then.